subprojects/packagefiles/speexdsp/meson_options.txt

```diff
@@ -0,0 +1,2 @@
+option('use-fixed-point', type : 'boolean', value : false, description : 'Compile fixed-point library')
+option('with-example-programs', type : 'boolean', value : false, description : 'Build example programs')
```

Comment:
Maybe these option names could be simplified to e.g. `fixed-point` and `examples`? The "with" feels a bit redundant in particular.

Reply:
Changed, thanks!